docs/sdk/dev/telemetry.md

```diff
@@ -42,13 +43,22 @@ This interface uses `SprykerSdk\Sdk\Core\Application\Dependency\Service\Telemetr
 To implement the custom sender, you need to implement this interface and make sure to throw `SprykerSdk\Sdk\Core\Application\Exception\TelemetryServerUnreachableException` when the destination server is unreachable.
 
 ## Task console event collecting
-Only task commands are collected. This behavior is implemented by listening to the generic console events. 
+Currently, only task commands are collected. This behavior is implemented by listening to the generic console events.
```

Comment:
Please avoid the word "currently" in docs. Check here:  https://developers.google.com/style/timeless-documentation